assembly/src/parsers/u32_ops.rs

```diff
@@ -152,37 +160,39 @@ pub fn parse_u32split(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), A
 /// inserted. Please refer to the docs of `handle_arithmetic_operation` for more details.
 ///
 /// VM cycles per mode:
-/// - u32add: 4 cycles
-/// - u32add.b: 5 cycles
-/// - u32add.full: 2 cycles
-/// - u32add.unsafe: 1 cycle
-pub fn parse_u32add(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), AssemblyError> {
-    handle_arithmetic_operation(span_ops, op, Operation::U32add)
+/// - u32checked_add: 4 cycles
+/// - u32checked_add.b:
+///    - 6 cycles if b is 1
+///    - 5 cycles if b is not 1
```

Comment:
Minor nit: would prefer to use `b = 1` and `b != 1` instead of words here and in other places.